HW_1/task_2.py

```diff
@@ -0,0 +1,13 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов.
+# Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов,
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+sum = int(input('Введите сумму сделанных журавликов, кратную 6: '))
+while (sum % 6 != 0):
+    sum = int(input('Введены не корректные данные.'))
+else:
+    print((sum // 6), (( sum // 6) * 4), (sum // 6))
```

Comment:
![image](https://user-images.githubusercontent.com/49741194/231382055-c23659f3-ac01-4d8a-9097-c40fe99678fe.png)
```suggestion
    print(sum // 6, sum // 6 * 4, sum // 6)
```

Reply:
А еще проще поместить повторяемое выражение в переменную, чтобы постоянно не выделять ресурсы вычислительной машины на его вычисление
```suggestion
    res = sum // 6
    print(res, res  * 4, res)
```